library/src/main/java/com/joom/grip/io/JrtFileSource.kt

```diff
@@ -0,0 +1,21 @@
+package com.joom.grip.io
+
+import java.nio.file.Files
+import java.nio.file.Path
+import kotlin.io.path.extension
+
+class JrtFileSource(private val jrtPath: Path) : FileSource {
```

Comment:
And how is it different from `DirectoryFileSource`?

Reply:
Well it pretty close actually, but under the hood Paths.get("jrt:/...") will create instance of `JrtPath`, instead of `UnixPath` for example. Implementation of this classes is different, for example `JrtPath.toFile` will throw `UnsupportedOperationException`